src/PuJetIdSF.cc

```diff
@@ -280,8 +200,7 @@ std::vector<float> PuJetIdSF::getEvtWeight(bigTree &theBigTree, TLorentzVector t
 
     if (DEBUG) std::cout << "   isRealJet: " << isRealJet << " passPUjetIDLoose: " << passPUjetIDLoose << std::endl;
 
-    // Get Efficiency and SF
-    float eff = getEff(isRealJet, tlv_jet.Pt(), tlv_jet.Eta());
+    // Get SF
     float sf  = getSF (isRealJet, tlv_jet.Pt(), tlv_jet.Eta());
 
```

Comment:
I propose to remove the getEff function also from the .h file and to modify the getSF one removing the isReaJet flag since we need to apply scale factors only for real jets.

Reply:
I did not remove the `getEff` function at all, is that what you're suggesting (ie from the `.cc` file as well)? regarding the second part of the comment (also answering the other comment): it might be helpful for readability, so why not, but note that it would not change anything in terms of the "event-level" SF value

Reply:
Yep, I propose to remove the getEff method from the .cc and .h file since it is not needed. And yes I know that removing the isRealJet flag won't change the SF value but I think it is not needed.

Reply:
done